src/bin/tikv-ctl.rs

```diff
@@ -877,9 +878,11 @@ impl DebugExecutor for Debugger {
 
 fn main() {
     let raw_key_hint: &'static str = "raw key (generally starts with \"z\") in escaped form";
+    let version_info = util::tikv_version_info();
 
     let mut app = App::new("TiKV Ctl")
-        .author("PingCAP")
+        .long_version(version_info.as_ref())
+        .author("PingCAP Inc. <info@pingcap.com>")
```

Comment:
What about `The TiKV Project Developers`? What do you think @siddontang ?

Reply:
Seems it's just the same as that in tikv-server.rs. Maybe it's better to extract it as one global string.

Reply:
TiKV Org.

No need email here.

Reply:
Ok.